cmd/embedded-cluster/uninstall.go

```diff
@@ -402,9 +402,6 @@ var resetCommand = &cli.Command{
 			if err := os.RemoveAll(lamPath); err != nil {
 				return err
 			}
-			if err := os.RemoveAll(defaults.LocalArtifactMirrorPath()); err != nil {
-				return err
-			}
```

Comment:
shouldn't we still remove this?

Reply:
oops I see we changed the location, never mind

Reply:
We were moving the binary to `/usr/local/bin` and using it from there. Now we are using the binary from `/var/lib/embedded-cluster/bin` directly and the whole directory is already removed during resets.